src/System.Text.Json/tests/Serialization/SpanTests.cs

```diff
@@ -32,8 +32,9 @@ public static void ReadGenericApi()
         }
 
         [Fact]
-        public static void VerifyValueFail()
+        public static void ToStringNullArgumentFail()
         {
+            Assert.Throws<ArgumentNullException>(() => JsonSerializer.ToString((object)null));
```

Comment:
Same here, this is failing:
https://mc.dot.net/#/user/dotnet-bot/pr~2Fdotnet~2Fcorefx~2Frefs~2Fpull~2F36201~2Fmerge/test~2Ffunctional~2Fcli~2F/20190320.37/workItem/System.Text.Json.Tests/analysis/xunit/System.Text.Json.Serialization.Tests.SpanTests~2FToStringNullArgumentFail
```text
Message :
Assert.Throws() Failure
Expected: typeof(System.ArgumentNullException)
Actual:   (No exception was thrown)
Stack Trace :
   at System.Text.Json.Serialization.Tests.SpanTests.ToStringNullArgumentFail() in /__w/1/s/src/System.Text.Json/tests/Serialization/SpanTests.cs:line 38
```

Reply:
To be clear, not this line but the next one (i.e. 38). At least that's what the log says.